libraries/AP_Compass/AP_Compass_AK09916.cpp

```diff
@@ -170,6 +170,7 @@ AP_Compass_Backend *AP_Compass_AK09916::probe_ICM20948(AP_HAL::OwnPtr<AP_HAL::I2
 AP_Compass_Backend *AP_Compass_AK09916::probe_ICM20948(uint8_t inv2_instance,
                                                      enum Rotation rotation)
 {
+#if HAL_INS_ENABLED
```

Comment:
is it not possible to compile these out from frontend directly? And wrap entire sources of backends with this?

Reply:
ohhh... I see what you are doing, nevermind.

Reply:
So.. it's OK? Approved?